b2b_ecommerce/models.py

```diff
@@ -16,6 +18,42 @@
 from mitxpro.utils import serialize_model_object
 
 
+class B2BCoupon(TimestampedModel, AuditableModel):
+    """
+    A coupon for B2B purchases
+    """
+
+    coupon_code = models.CharField(max_length=50)
+    discount_percent = models.DecimalField(
+        decimal_places=2,
```

Comment:
If you let me do more decimal places, then I can craft a discount coupon for any particular price, instead of getting stuck with pretty percentages and ugly prices.

Reply:
Sounds good, how many do you think would be sufficient?

Reply:
I feel like there should be a mathematically provable answer to this, but I'm lazy. Using guess and check, I think 5 digits is sufficient.

Reply:
:+1: